src/Kunstmaan/FormBundle/Form/AbstractFormPageAdminType.php

```diff
@@ -28,13 +30,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'class' => 'js-rich-editor rich-editor'
             ),
         ));
-        $builder->add('subject', null, array(
+        $builder->add('subject', TextType::class, array(
             'label' => 'kuma_form.form.page_admin.subject.label',
         ));
-        $builder->add('from_email', null, array(
+        $builder->add('from_email', EmailType::class, array(
             'label' => 'kuma_form.form.page_admin.from_email.label',
         ));
-        $builder->add('to_email', null, array(
+        $builder->add('to_email', EmailType::class, array(
```

Comment:
This should be texttype? Currently it is possible to pass a comma delimited string of emails